src/main/java/gameframework/base/ObservableValue.java

```diff
@@ -2,13 +2,31 @@
 
 import java.util.Observable;
 
+/**
+ * Class that represent an observable value.
+ * Observable values are ideal to "trigger events"
```

Comment:
Why "ideal"? It's their sole purpose; instead, why not explain how/why/where it's useful in the framework, and give a schematic usage example?

The quotes around _trigger events_ are unneeded, and this comment could mention or link to the Observer design pattern.